### PR TITLE
[Java] support python worker command in raylet

### PR DESCRIPTION
**Pull request by chuxi**
## What do these changes do?

support raylet, which is started by java runManager, to start python default_worker.py . 

So when doing local test of java call python task, it helps auto start python worker.

### CODE REVIEW

**Comment by chuxi** [comment]
... @AmplabJenkins restart? 

How to apply the privilege to restart the failed ci test? ;(

**Comment by raulchen** [comment]
@chuxi you can close and reopen this PR to trigger CI

**Review by raulchen — APPROVED** [review]

**Comment by raulchen** [comment]
@chuxi you can close and reopen this PR to trigger CI

**Comment by chuxi** [comment]
@raulchen  get it. Thank you!

**Comment by raulchen** [comment]
hmm, weird that the last CI job kept failing... let me try again

**Comment by raulchen** [comment]
hmm, weird that the last CI job kept failing... let me try again

**Comment by chuxi** [comment]
@raulchen  ... 

why you always post two same comments ... :p 

The ray test is broken actually. I will rebase master and push again.

**Comment by chuxi** [comment]
@robertnishihara 

... so is the master branch ci broken? I did not change any code except java, but the failed tests are python's ... :(

**Comment by robertnishihara** [comment]
The Jenkins failure should be fixed in https://github.com/ray-project/ray/pull/3112.  Some of the Travis tests are a little flakey.

**Comment by chuxi** [comment]
rerun failed test or merge directly ?
